### ISSUE TITLE
Reintroduce the CDN version

### ISSUE THREAD

**Issue by kossnocorp** [date-fns/date-fns#1780]
Developers that just start and those who don't use any browser bundler struggle with using date-fns. It's a shame to force them to use Moment.js, so I think it worth reintroducing the CDN version.

**Comment by RavsCodes20** [comment]
Struggling to use a browser bundler myself. I think a CDN version would be very helpful in getting pages off the ground and allowing for quicker development. Don't really like having to mess with bundling myself if I can avoid it.b

**Comment by maxwell8888** [comment]
I find it really useful not having to use a bundler from time to time, for example for quickly playing around with something. You could also look I providing an UNPKG link, I tried the below but I'm getting CORS errors for some reason:
```
import {
        format,
        eachDayOfInterval,
        differenceInDays,
      } from "https://unpkg.com/date-fns?module";
```

**Comment by xavierrobitaille** [comment]
another perk of the CDN version is that it lets you use the chrome console as a sandbox to play around with dateFns:
`var script = document.createElement("script");
script.src = "https://cdnjs.cloudflare.com/ajax/libs/date-fns/1.30.1/date_fns.js";
document.body.appendChild(script);
`
then
`dateFns.parse('06/04/2020');
//Thu Jun 04 2020 00:00:00 GMT-0400 (Eastern Daylight Time)`

**Comment by MattIPv4** [comment]
Hi from cdnjs,

We'd love to see a bundled version of this library provided again. The last version we currently have is 1.30.1, and we have users who are asking for new versions to be added, but we can't as there is no bundled version in the NPM package.

cc https://github.com/cdnjs/packages/issues/392

**Comment by ro-savage** [comment]
I would like to use date-fns in a simple chrome extension. But can't no unless I set up an entire bundling process.

It feels crazy that this option was removed. Is the a reason?

**Comment by maupin** [comment]
I'll add my voice to those asking for a CDN version.  Not having one is a huge hurdle for beginners to clear.

**Comment by conundrumer** [comment]
Doesn't look like it'll happen: https://github.com/date-fns/date-fns/issues/276#issuecomment-330791697

**Comment by kossnocorp** [comment]
@conundrumer it will happen 😉

**Comment by MartinKolarik** [comment]
We recently added a new feature at jsDelivr which makes this (and other ES modules) work without a build step. [Small demo here](https://jsfiddle.net/1cazpb5t/) and [more details here](https://esm.run/).

**Comment by michaelBenin** [comment]
@MartinKolarik is it possible to get a url for that prebuilt lib?

**Comment by MartinKolarik** [comment]
@michaelBenin please check the demo and docs in my previous comment.

**Comment by FredKSchott** [comment]
Hey, big fan of date-fns at Skypack! We include it as our "recommended" package when you search for anything like date or time on the Skypack catalog: https://www.skypack.dev/search?q=datetime&p=1. Keep up the good work!

This package works today on the Skypack CDN! No action needed on your end. It's actually already been requested and used 2,500+ times in the last 7 days.

More info: https://www.skypack.dev/view/date-fns

```js
// Example
import { /* ... */ } from 'https://cdn.skypack.dev/date-fns';
```

**Comment by smac89** [comment]
Hey can you guys also update your website to include these new methods? It would really help!

Thanks!

**Comment by hughsheehy** [comment]
> Hey, big fan of date-fns at Skypack! We include it as our "recommended" package when you search for anything like date or time on the Skypack catalog: https://www.skypack.dev/search?q=datetime&p=1. Keep up the good work!
> 
> This package works today on the Skypack CDN! No action needed on your end. It's actually already been requested and used 2,500+ times in the last 7 days.
> 
> More info: https://www.skypack.dev/view/date-fns
> 
> ```js
> // Example
> import { /* ... */ } from 'https://cdn.skypack.dev/date-fns';
> ```

Thanks for that. Working on a proto and had already built all the time functions with date-fns when we thought about just statically hosting the whole app so the CDN version was a lifesaver. Amazed it's not included within the package itself. Is there anything anyone outside the project can do? To help?

**Comment by hughsheehy** [comment]
> @michaelBenin please check the demo (it shows specifically date-fns) and docs in my previous comment. It won't create a global though. We preserve the ES format, just bundle to make it work in the browser ([browser support here](https://caniuse.com/es6-module)).

When I look for the browser support, date-fns isn't there. Am I missing something?

**Comment by rosdi** [comment]
Yeah... I had to bundle this to make it work. I prefer not to bundle and use CDN instead. Any particular reason why CDN support was removed? Too much effort required to maintain it?

My project is a multiple-spa project, it is an ASP.NET Core project that uses Vue for frontend scripting. Each pages is a SPA of their own. So it does not make sense to bundle this in every pages. A CDN link would be much more efficient.

Anyway, I think I will just use momentjs since it supports CDN.

**Comment by cachius** [comment]
For v3 CDN support is planned! → https://twitter.com/kossnocorp/status/1634892171554832384

**Comment by LeeKorbis** [comment]
The reason why a browser-ready version is necessary is because there are cases where a bundler is not necessary. Pretty simple reasoning.

For us, we produce HTML and include <script src="..."> for things that are needed. Our build process is very simple thanks to not needing webpack to support older browsers. We use a simple Makefile.

Thank you for an excellent date-time library and hopefully this can be resolved!

**Comment by maxilirator** [comment]
For others new to this, dayjs with CDN was much easier to get quick and dirty working.

**Comment by eithe** [comment]
Did CDN support land for v3? Wanted to include date-fns in a legacy app and found https://www.jsdelivr.com/package/npm/date-fns which says v3.

**Comment by kossnocorp** [comment]
The CDN version is now available (starting with `date-fns@3.6.0`), see the guide: https://date-fns.org/v3.6.0/docs/CDN
